site/app/_components/AuthorLink.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+interface AuthorLinkProps {
+  handle: string;
+}
+
+const AuthorLink: React.FC<AuthorLinkProps> = ({ handle }) => {
+  const username = handle.replace('@', '');
+  const githubUrl = `https://github.com/${username}`;
+
+  return (
+    <a href={githubUrl} target="_blank" rel="noopener noreferrer">
+      {handle}
+    </a>
+  );
+};
+
+export default AuthorLink;
```

Comment:
```suggestion
export default AuthorLink;

```

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline